room.go

```diff
@@ -997,7 +1006,7 @@ func (r *Room) handleIncomingRpcRequest(callerIdentity, requestId, method, paylo
 		if _, ok := err.(*RpcError); ok {
 			r.engine.publishRpcResponse(callerIdentity, requestId, nil, err.(*RpcError))
 		} else {
-			r.engine.log.Warnw("Uncaught error returned by RPC handler for method, using application error instead", err, "method", method)
+			r.log.Warnw("Uncaught error returned by RPC handler for method, using application error instead", err, "method", method)
```

Comment:
:nit: lower case `u` in `Uncaught`, our convention is to use lower case in logging.